app/views/config_groups/index.html.erb

```diff
@@ -1,7 +1,8 @@
 <% title _('Config Groups') %>
 
 <% title_actions new_link(_('Create Config Group')), help_button %>
-
+<%= alert :header => '', :class => 'alert-warning', :close => false,
+          :text => _("Managing config groups is being extracted to the foreman_puppet_enc plugin and will be removed from core in 3.0") %>
```

Comment:
```suggestion
          :text => _("Config Groups are being extracted to the foreman_puppet plugin and will be removed from core in 3.0") %>
```
This is not managing of cfg groups, as it is concept in Foreman, not just a management of resource in puppet.

Reply:
Another option as Ewoud correctly noted, this is all about Puppet Classes.
```suggestion
          :text => _("Puppet Classes management is being extracted to the foreman_puppet plugin and will be removed from core in 3.0") %>
```